usecases-integration/src/test/java/eu/europeana/cloud/integration/usecases/RepresentationRevisionFilesTestCase.java

```diff
@@ -0,0 +1,120 @@
+            Revision revision = revisions.get(0);
+
+            List<CloudTagsResponse> cloudTagsResponses = sourceDatasetHelper.getDataSetRevisions(SOURCE_PROVIDER_ID, SOURCE_DATASET_NAME, SOURCE_REPRESENTATION_NAME, revision.getRevisionName(), revision.getRevisionProviderId(), TestHelper.getUTCDateString(revision.getCreationTimeStamp()));
+            assertNotNull(cloudTagsResponses);
+            assertEquals(cloudTagsResponses.size(), 1);
+            String responseCloudId = cloudTagsResponses.get(0).getCloudId();
+            assertEquals(responseCloudId, cloudId);
+
+            RepresentationRevisionResponse representationRevisionResponse = sourceRecordServiceClient.getRepresentationRevision(responseCloudId, SOURCE_REPRESENTATION_NAME, revision.getRevisionName(), revision.getRevisionProviderId(), TestHelper.getUTCDateString(revision.getCreationTimeStamp()));
+            assertNotNull(representationRevisionResponse.getFiles());
+            assertEquals(representationRevisionResponse.getFiles().size(), 1);
+
+            InputStream stream = sourceFileServiceClient.getFile(representationRevisionResponse.getCloudId(), SOURCE_REPRESENTATION_NAME, representationRevisionResponse.getVersion(), representationRevisionResponse.getFiles().get(0).getFileName());
+            List<String> lines = IOUtils.readLines(stream);
+            assertNotNull(lines);
+            assertEquals(lines.size(), 1);
+            assertEquals(lines.get(0), FILE_CONTENT);
+            //Do any Conversion to the file content; Mock it in here to use the same content
+            String convertedContent = lines.get(0);
+
+            String uri = sourceDatasetHelper.addFileToNewRepresentation(DESTINATION_REPRESENTATION_NAME, SOURCE_PROVIDER_ID, convertedContent);
+            assertNotNull(uri);
+            String version = sourceDatasetHelper.getVersionFromFileUri(uri);
+            sourceDatasetHelper.addRevision(SOURCE_PROVIDER_ID, DESTINATION_REPRESENTATION_NAME, DESTINATION_REVISION_NAME, tags, responseCloudId, version);
+            revisions = getRevisions(responseCloudId, DESTINATION_REPRESENTATION_NAME);
+            assertNotNull(revisions);
+            assertEquals(revisions.size(), 1);
+            assertEquals(revisions.get(0).getRevisionName(), DESTINATION_REVISION_NAME);
+
+            System.out.println("RepresentationRevisionFilesTestCase Finished Successfully ..");
+
+        } finally {
+            cleanUp();
+        }
+    }
+
+    private List<Revision> getRevisions(String cloudId, String representationName) throws MCSException {
+        Representation representation = sourceRecordServiceClient.getRepresentation(cloudId, representationName);
+        return representation.getRevisions();
+    }
+
+
+    private void prepareTestCase(String providerId, String datasetName, String representationName, String revisionName, List<String> tags, int recordNum, String cloudId) throws MCSException, MalformedURLException, CloudException {
```

Comment:
What is the purpose of this private method? It does not do anything with passed parameters, it is used only once in the code. There is no need for this method in my opinion.